plugins/module_utils/aci.py

```diff
@@ -454,7 +463,10 @@ def cert_auth(self, path=None, payload="", method=None):
     def response_json(self, rawoutput):
         """Handle APIC JSON response output"""
         try:
-            jsondata = json.loads(rawoutput)
+            if isinstance(rawoutput, dict):
+                jsondata = rawoutput
```

Comment:
In what situation will this ever be a dict? 
Should we split the function to limit to one specific task?

Reply:
This was part of the old code (a couple of years ago). I'll remove it